inlineplz/interfaces/github.py

```diff
@@ -208,6 +211,17 @@ def post_messages(self, messages, max_comments):
             print("Github connection is invalid.")
             return
 
+        if self.autofix and git.files_changed() and not self.out_of_date():
+            print("Files changed: attempting to push fixes")
+            for filename in self.filenames:
+                print("Adding {}".format(filename))
+                git.add(filename)
+            git.commit("Autofix by inline-plz")
```

Comment:
Request: please make the git commit message follow the guidelines for good commit messages: https://chris.beams.io/posts/git-commit/

Reply:
hmmm do you have a suggested commit message phrase?

Reply:
Hmm... maybe
```
Fix formatting issues

This commit was automatically created by inline-plz.
```

Reply:
Or, if we can't do full commit messages, maybe `Fix formatting issues (according to inline-plz)`

Reply:
hmm i think autofix stuff will usually be formatting, but it's possible there are tools that can autofix code beyond just formatting.